AjuntamentdeBarcelona/decidim.barcelona
# Meeting show page

# What and why

I have created the meetings show page with basic information. I reused the `StaticMap` component to render a google maps.
The sidebar displays a sample of 3 related proposals, I'm not sure if that should be randomized or not.

Finally, I haven't include anything related to `categories` and `subcategories` because we are waiting for the graphical design.
# QA

Visit a meeting page from the meetings directory. Verify the basic information and if the map is working properly.
# GIF Tax

![](https://media.giphy.com/media/adhuPKADfzrDG/giphy.gif)
